google/cloud/storage/client.h

```diff
@@ -333,13 +333,14 @@ class Client {
    * @par Example
    * @snippet storage_bucket_samples.cc create bucket
    *
-   * @see Before enabling Bucket Policy Only please review the
-   *     [feature documentation][bpo-link], as well as
-   *     ["Should you use Bucket Policy Only?"][bpo-should-link].
+   * @see Before enabling Uniform Bucket Level Access please review the
+   *     [feature documentation][ubla-link], as well as
+   *     ["Should you use Uniform bucket-level access ?"][ubla-should-link].
    *
-   * [bpo-link]: https://cloud.google.com/storage/docs/bucket-policy-only
-   * [bpo-should-link]:
-   * https://cloud.google.com/storage/docs/bucket-policy-only#should-you-use
+   * [ubla-link]:
```

Comment:
the link appears in the next line. I think the generated document will break due to this. We must have the link on the same line.

Reply:
Hi Manish
When the links are moved to the next line, docker build throws complains about formatting. Please refer attached screenshot image.
![docker_build_3193](https://user-images.githubusercontent.com/71334750/95304939-f569ae00-08a2-11eb-989d-1b74d48ccba3.png)

And when the links are on the same line, docker build doesn't through any formatting complain.

Please suggest.